test/metadata_service/endpoint_mode_config_options.spec.js

```diff
@@ -0,0 +1,50 @@
+var helpers = require('./../helpers');
+var AWS = helpers.AWS;
+
+var options = require('../../lib/metadata_service/endpoint_mode_config_options');
+var ENDPOINT_MODE_CONFIG_OPTIONS = options.ENDPOINT_MODE_CONFIG_OPTIONS;
+var ENV_ENDPOINT_MODE_NAME = options.ENV_ENDPOINT_MODE_NAME;
+var CONFIG_ENDPOINT_MODE_NAME = options.CONFIG_ENDPOINT_MODE_NAME;
+
+var EndpointMode = require('../../lib/metadata_service/endpoint_mode');
```

Comment:
Move import statement to top?

Reply:
Done in [`5a8e107` (#3853)](https://github.com/aws/aws-sdk-js/pull/3853/commits/5a8e10755d8a9b04183b19f5611bb4b0b5bbf137)